localhost.env

```diff
@@ -9,3 +9,17 @@ DB_DRIVER=org.postgresql.Driver
 COMPOSE_ENABLED=true
 COMPOSE_PROFILE=development
 COMPOSE_LIFECYCLE=start-and-stop
+
+# Application configuration
+APPLICATION_URL=http://localhost:8080
+JACKSON_NAMING=SNAKE_CASE
+
+# Mail configuration
+MAIL_HOST=sandbox.smtp.mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=c9c80f4256e023
+MAIL_PASSWORD=4bf3255b5aaa02
+MAIL_PROTOCOL=smtp
+MAIL_TEST_CONNECTIONS=true
+MAIL_SMTP_AUTH=true
+MAIL_START_TLS_ENABLE=true
```

Comment:
Please, add these variables to the `localhost.run.xml` file as well :)

Reply:
Done😊